docs/07-web-development.Rmd

```diff
@@ -350,8 +350,52 @@ It is not a perfect solution, but it should be very rare that you run into probl
 
 ## Useful resources
 
+### Optimization
+
 Although\index{Optimization} static websites are fast in general, you can certainly further optimize them. You may search for "CSS and JavaScript minifier", and these tools can compress your CSS and JavaScript files, so that they can be loaded faster. Since there are a lot of tools, I will not recommend any here.
 
 You can also optimize images on your website. I frequently use a command-line tool named [`optipng`](http://optipng.sourceforge.net) to optimize PNG images. It is a lossless optimizer, meaning that it reduces the file size of a PNG image without loss of quality. From my experience, it works very well on PNG images generated from R, and can reduce the file size by at least 30% (sometimes even more than 50%). Personally I also use online tools like http://optimizilla.com to optimize PNG and JPEG images. For GIF images, I often use https://ezgif.com/optimize to reduce the file sizes if they are too big.
 
 Note that Netlify has provided the optimization features on the server side for free at the moment, so you may just want to enable them there instead of doing all the hard work by yourself.
+
+### blogdown resources
+
+We have created a minimal **blogdown** example website to demonstrate how to use R Markdown features (https://blogdown-demo.rbind.io). Some example posts are:
+
+- [How to add cites in posts](https://blogdown-demo.rbind.io/2017/08/28/adding-citations-to-posts/)
+- [How to add R code to posts](https://blogdown-demo.rbind.io/2017/10/06/adding-r-code-to-r-markdown-posts/)
+- [How to include figures and other images in posts](https://blogdown-demo.rbind.io/2017/10/06/adding-figures-and-other-images-to-posts/)
+
+### Hugo resources
+
+Here we provide some useful links for beginning Hugo and **blogdown** users:
+
+- [Hugo directory structure](https://gohugo.io/getting-started/directory-structure/) explains the basics of what goes where in a Hugo project
```

Comment:
This style `[text](link) does this that` is good for HTML output but can be confusing to readers when the output format is PDF (e.g. those who read the hardcopies of this book). This is because on web pages, readers can see these links (in blue), but in PDF, you won't see any differences between the link and the following text. This sentence will read like this:

> Hugo directory structure explains the basics of what goes where in a Hugo project

Could you come up with a different way to present these links?

BTW, there should be punctuations after each bullet (semicolon or period).

Reply:
I tried tables- let me know what you think!